### PR TITLE
Clickable event card

### PR DESCRIPTION
**Pull request by sdotson**
This is addressing issue #56 . Open to other solutions! One though I had is that it might be helpful to add a hover style, perhaps a background-color, so users know that something is going to happen if they click the list item.

### CODE REVIEW

File: src/components/EventCard/EventCard.js [hunk 1]
```diff
@@ -25,25 +25,25 @@ const EventCard = ({ event, className }) => {
...
-        <img
-          src={featuredImageUrl || 'http://www.fillmurray.com/550/450'}
-          alt="featured event"
-        />
-      </div>
-      <div className={styles.contentWrapper}>
-        <div className={styles.dateLocation}>
-          <DateBlock date={startDate} />
-          { renderLocation(loc) }
+    <Link to={`/event/${_id}`}>
```
Comment (pdw207) [inline src/components/EventCard/EventCard.js]: You can't have an `li` inside an `a` tag.  Just change the nesting and `ul li a { display:block; }`

Comment (InciteDemocracy) [inline src/components/EventCard/EventCard.js]: @sdotson A slight darkening of the background on hover sounds like it could work nicely!

**Comment by sdotson** [comment]
- added a hover bg color to the li
- moved anchor back to inside li
- fixed styles of li > a
- moved li padding to a (so that the whole li area would be clickable and not have that awkward 35px dead zone)

Take a look. Happy to receive further feedback.

**Comment by sdotson** [comment]
Went ahead and fixed merge conflict.

**Pull request merged**